nulib/dc-nextjs
# Wire Honeybadger into the app

## What does this do?
Adds Honeybadger code to the Next app.  @mbklein added the supporting Terraform vars to staging and production.

## How to test?
1. In a local dev environment, throw a `new Error("ima error")` in some component. 
2. Log into https://app.honeybadger.io/projects and view the `dc-next` project and the error should appear.
